fimptype/primefimp/api-client.go

```diff
@@ -340,7 +322,25 @@ func (mh *ApiClient) sendGetRequest(components []string) (*fimpgo.FimpMessage, e
 	mh.sClient.AddSubscription(respAddr.Serialize())
 
 	param := RequestParam{Components: components}
-	req := Request{Cmd: CmdGet, Param: param}
+	req := Request{Cmd: CmdGet, Param: &param}
+
+	msg := fimpgo.NewMessage("cmd.pd7.request", "vinculum", fimpgo.VTypeObject, req, nil, nil, nil)
+	msg.ResponseToTopic = respAddr.Serialize()
+	msg.Source = mh.clientID
+	return mh.sClient.SendFimpWithTopicResponse(reqAddr.Serialize(), msg, respAddr.Serialize(), "", "", 5)
+}
+
+func (mh *ApiClient) sendSetRequest(component string, value interface{}) (*fimpgo.FimpMessage, error) {
+	reqAddr := fimpgo.Address{MsgType: fimpgo.MsgTypeCmd, ResourceType: fimpgo.ResourceTypeApp, ResourceName: "vinculum", ResourceAddress: "1"}
+	respAddr := fimpgo.Address{MsgType: fimpgo.MsgTypeRsp, ResourceType: fimpgo.ResourceTypeApp, ResourceName: mh.clientID, ResourceAddress: "1"}
+	if mh.cloudService != "" {
+		respAddr.ResourceType = fimpgo.ResourceTypeCloud
+		respAddr.ResourceName = "backend-service"
+		respAddr.ResourceAddress = mh.cloudService
+	}
+	mh.sClient.AddSubscription(respAddr.Serialize())
```

Comment:
Better to move AddSubsription to the constructor. Otherwise, the lib will be "subscribing"  on every command.